src/Compiler/CompilerUtils.cpp

```diff
@@ -251,6 +237,20 @@ struct Command {
 // =============================================================================
 // Methods for compiling and file processing.
 
+std::string getOnnxMlirFullVersion() {
+  const std::string OnnxMlirVersion = "onnx-mlir version 0.3.0";
```

Comment:
If I recall correctly there was a separate change committed recently that added the option to specify the onnx-mlir version. I would not expect to see hardcoded versions here any more, was this missed last time?

Reply:
I don't recall such change.

Reply:
https://github.com/onnx/onnx-mlir/pull/1409

Reply:
Product version is different from onnx-mlir version.

Reply:
How so? I would expect that they are the same - when would they be different?

This is why we need a single proposal - there are too many changes impacting how the version is printed, what the version is, etc. and it is hard to keep track of what they are and what the goal is. A single clear goal will reduce confusion significantly and will let you get buy off from everyone (or at least everyone will know what to expect).

Reply:
Yes, I am working on that.

onnx-mlir version is the version for the Open Source project onnx-mlir.
Product version is the version of the product a company may want to ship, where onnx-mlir is one of the components.

Reply:
https://github.com/onnx/onnx-mlir/issues/1460 is created for the discussion. @sstamenova Please let me know if it is more clear.